bobtemplates/plone/view.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+"""Generate view."""
+
+from bobtemplates.plone.base import base_prepare_renderer
+from bobtemplates.plone.base import git_commit
+from bobtemplates.plone.base import update_file
+from lxml import etree
+from mrbob.bobexceptions import SkipQuestion
+from mrbob.bobexceptions import ValidationError
+
+import os
+import stringcase
+
+
+def get_view_name(configurator, question):
```

Comment:
you describe in the doc string way better what the method is doing.
Try to improve the method name, so that it makes more sense.